src/App.php

```diff
@@ -797,20 +812,51 @@ public function init_pagehead()
 			'$shortcut_icon'   => $shortcut_icon,
 			'$touch_icon'      => $touch_icon,
 			'$stylesheet'      => $stylesheet,
```

Comment:
shouldn't be `$stylesheet` the first entry of `$stylesheets`?

Reply:
I don't understand exactly what the `$stylesheet` template variable is supposed to hold, but one thing for sure is that it doesn't hold any URL that I can figure out.